semanticdb/metacp/src/main/scala/scala/meta/internal/classpath/ClasspathIndex.scala

```diff
@@ -13,11 +13,7 @@ import scala.meta.io.Classpath
 import scala.meta.internal.io.PathIO
 
 /** An index to lookup class directories and classfiles by their JVM names. */
-final class ClasspathIndex private (val dirs: collection.Map[String, Classdir]) {
-
-  override def toString: String = {
-    s"ClasspathIndex(${dirs.size} entries)"
-  }
+final class ClasspathIndex private (classpath: Classpath, dirs: collection.Map[String, Classdir]) {
```

Comment:
I propose we expose the ability to query into `dirs` somehow, either make it a val or add a method `getClassdir(path: String): Option[Classdir]`. Making it private makes it impossible to investigate problems such as the bugs I discovered in https://github.com/scalameta/scalameta/pull/169

Reply:
Discussion in https://github.com/scalameta/scalameta/issues/1729.